src/dispatch/plugins/dispatch_slack/case/interactive.py

```diff
@@ -1245,6 +1245,14 @@ def triage_button_click(
     case = case_service.get(db_session=db_session, case_id=context["subject"].id)
     case.status = CaseStatus.triage
     db_session.commit()
+    # we run the case status transition flow
+    case_flows.case_status_transition_flow_dispatcher(
+        case=case,
+        current_status=CaseStatus.triage,
+        db_session=db_session,
+        previous_status=case.status,
```

Comment:
Would this be overwritten by the line above it?

```python
    case.status = CaseStatus.triage
    db_session.commit()
```

So, it's always `triage` -> `triage`

Reply:
Ah, yeah let me change the order.